docs/native-code.md

```diff
@@ -0,0 +1,122 @@
+---
+id: native-code
+title: Working with native code on Windows
+---
+
+## What is a React Native for Windows app?
+
+A RNW app is a [Universal Windows Platform app](https://docs.microsoft.com/en-us/windows/uwp/get-started/universal-application-platform-guide) (aka UWP or Windows 10 app).

```

Comment:
```suggestion
A RNW app is a [Universal Windows Platform app](https://docs.microsoft.com/windows/uwp/get-started/universal-application-platform-guide) (aka UWP or Windows 10 app).

```

Throughout, please remove the language tokens from the URLs so that folks visiting from other locales get the right version of the sites.